joomla/joomla-cms
# [4.0] Start moving JS outside of template

Pull Request for Issue # .

### Summary of Changes

This is phase 1 of moving the JS outside of the `template.js` and putting the code in files where it belongs.

### Testing Instructions

1. Ensure sidebar still works in Atum
2. Ensure the toolbar is still sticky on page scroll

@dgt41 @Fedik - Leaving the code for the radio buttons there for now

@Quy Please ignore code style as we'll be doing this as a batch at a later date
